cli-test/spec/automated_test_cases/performance_test.rb

```
@@ -7,21 +7,21 @@
-    cd 'test-small-repo'
+    measure_time('mkdir tmp\\aruba\\test-small-repo')
+    Dir.chdir('tmp\\aruba\\test-small-repo')
 
     # Generate image repository
     system('python ../benchmark/generate_image_repo.py --output_dir ~/test-small-repo/Data/10k_images --num_images 10000 --num_dirs 10 --image_size 128 128')
 
     # Initialize the repository
     init_time = measure_time('oxen init')
     puts "oxen init command took: #{init_time} seconds"
-    expect(init_time).to be < 7.0
+    expect(init_time).to be < 7.0F
```

RuboCop (1.69.1)

[fatal] unexpected token tCONSTANT
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)